css/qmwp-search-correlations.css

```diff
@@ -0,0 +1,286 @@
+.fieldsContainer {
+    font-size:0;
+    max-width: 100%;
+    position: relative;
+}
+
+.searchsubmit{
+    padding:6px 20px !important;	
+    text-transform: uppercase  !important;
+    background-color:#F16645  !important;
+    color:#FFF  !important;      
+    border-bottom-right-radius:2px  !important;
+    border-top-right-radius:2px  !important;
+    border-bottom-left-radius:0px  !important;
+    border-top-left-radius:0px  !important;
+    font-size:14px  !important;
+    height: 34px  !important;
+    border: 1px solid #EFEFEF  !important;    
+    border-left:none !important;
+    vertical-align: bottom  !important;
+    margin-bottom: 10px;
+    margin-top: 0px !important;
+}
+
+.searchsubmit:active {
+    background-color:#d65a3d  !important;
+}
+
+.selectOutputAsType{
+    border:1px solid #EFEFEF;
+    padding: 6px 10px;  
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Using height with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/964746528/issues/source?bid=2836188&fileBranchId=2859501#l31)